lib/include/hse_ikvdb/param.h

```diff
@@ -73,6 +73,7 @@ enum param_type {
     PARAM_TYPE_U16,
     PARAM_TYPE_U32,
     PARAM_TYPE_U64,
+    PARAM_TYPE_DOUBLE,
```

Comment:
Move after `ENUM` since that seems to be where you've been putting the case labels.